examples/speaker_recognition/notebooks/Speaker_Recognition_an4.ipynb

```diff
@@ -0,0 +1,1998 @@
+            "[NeMo W 2020-05-11 02:21:19 graph_outputs:167] Setting unigue name of the default output port `label` produced in step 15 by `audiotospeechlabeldatalayer3` to `15_audiotospeechlabeldatalayer3_label`\n",
+            "[NeMo W 2020-05-11 02:21:19 graph_outputs:167] Setting unigue name of the default output port `label_length` produced in step 15 by `audiotospeechlabeldatalayer3` to `15_audiotospeechlabeldatalayer3_label_length`\n",
+            "[NeMo W 2020-05-11 02:21:19 graph_outputs:167] Setting unigue name of the default output port `processed_signal` produced in step 16 by `audiotomelspectrogrampreprocessor1` to `16_audiotomelspectrogrampreprocessor1_processed_signal`\n",
+            "[NeMo W 2020-05-11 02:21:19 graph_outputs:167] Setting unigue name of the default output port `processed_length` produced in step 16 by `audiotomelspectrogrampreprocessor1` to `16_audiotomelspectrogrampreprocessor1_processed_length`\n",
+            "[NeMo W 2020-05-11 02:21:19 graph_outputs:167] Setting unigue name of the default output port `outputs` produced in step 17 by `jasperencoder1` to `17_jasperencoder1_outputs`\n",
+            "[NeMo W 2020-05-11 02:21:19 graph_outputs:167] Setting unigue name of the default output port `encoded_lengths` produced in step 17 by `jasperencoder1` to `17_jasperencoder1_encoded_lengths`\n",
+            "[NeMo W 2020-05-11 02:21:19 graph_outputs:167] Setting unigue name of the default output port `logits` produced in step 18 by `jasperdecoderforspkrclass1` to `18_jasperdecoderforspkrclass1_logits`\n",
+            "[NeMo W 2020-05-11 02:21:19 graph_outputs:167] Setting unigue name of the default output port `embs` produced in step 18 by `jasperdecoderforspkrclass1` to `18_jasperdecoderforspkrclass1_embs`\n",
+            "[NeMo W 2020-05-11 02:21:19 graph_outputs:167] Setting unigue name of the default output port `loss` produced in step 19 by `crossentropylossnm1` to `19_crossentropylossnm1_loss`\n"
+          ],
+          "name": "stderr"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8m7dz1-usp1S",
+        "colab_type": "text"
+      },
+      "source": [
+        "# Creating CallBacks\n",
+        "\n",
+        "We would like to be able to monitor our model while it's training, so we use callbacks. In general, callbacks are functions that are called at specific intervals over the course of training or inference, such as at the start or end of every n iterations, epochs, etc. The callbacks we'll be using for this are the SimpleLossLoggerCallback, which reports the training loss (or another metric of your choosing, such as \\% accuracy for speaker recognition tasks), and the EvaluatorCallback, which regularly evaluates the model on the dev set. Both of these callbacks require you to pass in the tensors to be evaluated--these would be the final outputs of the training and eval DAGs above.\n",
```

Comment:
\\% shows up incorrectly.

Reply:
"such as % accuracy for speaker recognition tasks" ---> shows correctly in my notebook.